tests/test_account.py

```diff
@@ -0,0 +1,34 @@
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
+import pytest
```

Comment:
is there are reason you use pytest over unittest?
My only reason to prefer unittest is that it is built in.

Reply:
Not a particular personal reason, I chose it after looking at online opinions of `unittest` versus `pytest`. Personally, I don't have an attachment to either. It shouldn't be too hard to change from one to the other if you or anyone else has a strong preference towards `unittest` 🙂

Reply:
I think it's fine to use `pytest`

Reply:
Okay, then we should just use one. There are unittest mock imports in addition to pytest imports.

Reply:
Sounds good - for mocking, the `mock` package was added to be part of the `unittest` package but I realized the `pytest` devs made a pytest wrapper `pytest-mock` so I will convert them appropriately.